Src/PChecker/CheckerCore/CheckerConfiguration.cs

```diff
@@ -263,6 +276,12 @@ public int MaxSchedulingSteps
         /// </summary>
         public bool DisableEnvironmentExit;
 
+
+        /// <summary>
+        /// A list of interesting events that should be delayed.
+        /// </summary>
+        public HashSet<Type> InterestingEvents = new();
```

Comment:
What do you mean by events that must be delayed? How is this used in the overall story?